packages/developer-extension/src/background/domain/logRequests.ts

```diff
@@ -0,0 +1,64 @@
+import { store } from '../store'
+
+const decoder = new TextDecoder('utf-8')
+chrome.webRequest.onBeforeRequest.addListener(
+  (info) => {
+    if (!store.logEvents) {
```

Comment:
some inconsistencies between `logRequests` and `logEvents` namings, what about `logEventsFromRequests` instead?